config/default/manager_default_images.yaml

```diff
@@ -12,6 +12,4 @@ spec:
       - name: manager
         env:
         - name: OPENSTACKCLIENT_IMAGE_URL_DEFAULT
-          # TODO: this image causes syslog warning messages, see upstream https://bugs.launchpad.net/tripleo/+bug/1999553
```

Comment:
the bug is related to validation-lib which is not installed in the client container now so I've removed this.